shared/ui/core/src/test/java/com/ivy/ui/FormatMoneyUseCaseTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.ivy.ui
+
+import android.content.Context
+import androidx.test.platform.app.InstrumentationRegistry
+import com.ivy.domain.features.BoolFeature
+import com.ivy.domain.features.FeatureGroup
+import com.ivy.domain.features.Features
+import com.ivy.ui.time.DevicePreferences
+import io.kotest.common.runBlocking
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+import java.util.Locale
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+class FormatMoneyUseCaseTest(private val param: TestData) {
+
+    private val context: Context = InstrumentationRegistry.getInstrumentation().context
+    private val features = mockk<Features>()
+    private val devicePreferences = mockk<DevicePreferences>()
+
+    class TestData(
+        val givenInput: Double,
+        val showDecimal: BoolFeature,
```

Comment:
This should be just true/false boolen and the test should handle the setup in the // Given block

Reply:
Done